client/Makefile

```diff
@@ -1,10 +1,10 @@
 #Use the gcc compiler
 CC = gcc
 CFLAGS = -g
-DEPS = ../lib/marshal/marshal.o
+DEPS = ../lib/marshal/marshal.o client_api.o
 LDFLAGS =
 #Objects created by makefile
-OBJS = client-kv
+OBJS = client-kv client_api_test
```

Comment:
This should be `client_api` instead of `client_api_test` (so that the API actually gets compiled from the `all` target).

`client_api_test` should be a dependency of the `test` target, which should also be declared as `.PHONY` so it always runs